src/main/java/org/jenkinsci/test/acceptance/docker/fixtures/GitLabContainer.java

```diff
@@ -0,0 +1,115 @@
+    protected static final String REPO_DIR = "/home/gitlab/gitlabRepo";
+
+    private static final HttpClient client = HttpClient.newBuilder()
+                                                       .followRedirects(HttpClient.Redirect.NORMAL)
+                                                       .connectTimeout(Duration.ofSeconds(5))
+                                                       .build();
+
+    private static final ElasticTime time = new ElasticTime();
+
+    public String host() {
+        return ipBound(22);
+    }
+
+    public int port() {
+        return port(22);
+    }
+
+    public URL getURL() throws IOException {
+        // return new URL("http://" + host() + ":" + port());
+        return new URL("http://" + getIpAddress());
+    }
+
+    /** URL visible from the host. */
+    public String getRepoUrl() {
+        return "ssh://git@" + host() + ":" + port() + REPO_DIR;
+    }
+
+    @Deprecated
+    public String getRepoUrlInsideDocker() throws IOException {
+        return "ssh://git@" + getIpAddress() + REPO_DIR;
+    }
+
+    /**
+     * URL visible from other Docker containers.
+     * @param alias an alias for this container’s {@link #getCid} passed to {@code --link}
+     */
+    public String getRepoUrlInsideDocker(String alias) throws IOException {
+        return "ssh://git@" + alias + REPO_DIR;
+    }
+
+    public HttpResponse<String> createRepo(String repoName, String token) throws IOException {
+        try{
+            HttpRequest request = HttpRequest.newBuilder()
+                                             .uri(new URI("http://" + getIpAddress() + "/api/v4/projects"))
+                                             .header("Content-Type", "application/json")
+                                             .header("PRIVATE-TOKEN", token)
+                                             .POST(HttpRequest.BodyPublishers.ofString("{ \"name\": \"" + repoName + "\" }"))
+                                             .build();
+            HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
+            return response;
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void deleteRepo(String token, String repoName) throws IOException, GitLabApiException {
+        // get the project and delete the project
+        GitLabApi gitlabapi = new GitLabApi("http://" + getIpAddress(), token);
+        ProjectApi projApi = new ProjectApi(gitlabapi);
+
+        Project project = projApi.getProjects().stream().filter((proj -> repoName.equals(proj.getName()))).findAny().orElse(null);
+        projApi.deleteProject(project);
+    }
+
+    public void waitForReady(CapybaraPortingLayer p) {
+        int timeout = (int) time.milliseconds(200000);
+        p.waitFor().withMessage("Waiting for GitLab to come up")
+                .withTimeout(Duration.ofSeconds(timeout)) // GitLab starts in about 2 minutes
```

Comment:
would be so much more readable with something like

```suggestion
                .withTimeout(Duration.ofSeconds(60 * 2)) // GitLab starts in about 2 minutes
```
or 
suggestion
                .withTimeout(Duration.ofMinutes(2)) // GitLab starts in about 2 minutes
```